docs/DeveloperGuide.md

```diff
@@ -357,6 +357,14 @@ Preconditions: The student's details have been previously added to TutorAid.
     
 **Extensions**
 
+* 1a. TutorAid detects an error in the list command.
+
+    * 1a1. TutorAid displays an error message and requests the tutor to re-enter the command.
+    * 1a2. User re-enters the command.
+
+      Steps 1a1-1a2 are repeated until the data entered are correct.
```

Comment:
Maybe can rephrase this to 

`Steps 1a1-1a2 are repeated until the command entered are correct.` 

so that it better suits the use case. This can also be applied to some of use cases below.

Otherwise LGTM

Reply:
Fixed, thanks!